### PR TITLE
[Hotfix] Configure doc and dashboard links with env vars

### PR DESCRIPTION
**Pull request by ajithranka**
Resolves [AR-3377](https://team-1624093970686.atlassian.net/browse/AR-3377)

Adds env vars for doc and dashboard links. 

This will be merged to both dev and main.

I've made corresponding changes on Netlify and 1Password.

### CODE REVIEW

**Review by shrinathprabhu — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -1,7 +1,9 @@
+ARCANA_DASHBOARD=
+ARCANA_DOCS=
+BTW_SO_ENABLED=false
```
Comment (karthik-durai) [inline .env.example:3]: is the value for `BTW_SO_ENABLED` intentional in example file?

can we add `URL` in suffix of `ARCANA_DASHBOARD` and `ARCANA_DOCS`

**Review by karthik-durai — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (ajithranka) [reply .env.example:3]: Don't recall why we set it to false. Must have been causing an issue during development.

Added URL suffixes.

Comment (karthik-durai) [reply .env.example:3]: I mean example file usually don't have values

Comment (ajithranka) [reply .env.example]: Removed.

**Pull request merged**